CHANGELOG.next.asciidoc

```diff
@@ -101,9 +101,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix parsing issues with nested JSON payloads in Elasticsearch audit log fileset. {pull}22975[22975]
 - Rename `network.direction` values in crowdstrike/falcon to `ingress`/`egress`. {pull}23041[23041]
 - Rename `s3` input to `aws-s3` input. {pull}23469[23469]
+- Disable the option of running --machine-learning on its own. {pull}20241[20241]
+- Fix PANW field spelling "veredict" to "verdict" on event.action {pull}18808[18808]
+- Add support for GMT timezone offsets in `decode_cef`. {pull}20993[20993]
```

Comment:
Are all of these additions to CHANGELOG.next expected?

Reply:
Nope this are old, some mishap in a previous release, I'll remove them